install/installer/pkg/components/ws-proxy/configmap.go

```diff
@@ -26,8 +27,15 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	header := HostHeader
 	blobServeHost := fmt.Sprintf("ide.%s", ctx.Config.Domain)
 	gitpodInstallationHostName := ctx.Config.Domain
-	gitpodInstallationWorkspaceHostSuffix := fmt.Sprintf(".ws.%s", ctx.Config.Domain)
+
+	installationShortNameSuffix := ""
+	if ctx.Config.Metadata.InstallationShortname != "" && ctx.Config.Metadata.InstallationShortname != configv1.InstallationShortNameOldDefault {
+		installationShortNameSuffix = "-" + ctx.Config.Metadata.InstallationShortname
+	}
+
+	gitpodInstallationWorkspaceHostSuffix := fmt.Sprintf(".ws%s.%s", installationShortNameSuffix, ctx.Config.Domain)
```

Comment:
@sagor999 I think with this change, we can remove `ucfg.Workspace.WSProxy.GitpodInstallationWorkspaceHostSuffix`, `ucfg.Workspace.WSProxy.GitpodInstallationWorkspaceHostSuffixRegex`, `ucfg.Workspace.WSProxy.GitpodInstallationHostName`